closure-typing/kebabToCamelCase.js

```diff
@@ -1,5 +1,22 @@
-function kebabToCamelCase() {
-  // your code here
+function kebabToCamelCase(b) {
+    let initialArray = b.split("-");
+    let newArray = [];
+
+    //I am convinced that it is possible to skip the change of letter case for the first element of the array at the same time saving it.
+    for (let i=0; i < initialArray.length; i++){
+        newArray.push(initialArray[i].charAt(0).toUpperCase()+initialArray[i].slice(1));
```

Comment:
this expression is overcomplicated, make it simpler, create some variables, extract to a function, it will healp it be more readable